lib/matplotlib/offsetbox.py

```diff
@@ -114,7 +114,7 @@ def _get_packed_offsets(widths, total, sep, mode="fixed"):
         offsets = offsets_[:-1]
         return total, offsets
 
-    elif mode == "equal":
+    else:  # "equal"
```

Comment:
I don't know whether this is better. On the one hand, you don't need it. On the other hand, keeping the symmetry and stating the value explicitly (not only as a comment also has some appeal.

Execution-wise it makes no difference. There is however a difference in case the if-else-chain gets out of sync with the `check_in_list` above. When using `else` other values will be treated like "equal". When using `elif mode` other values don't get that execution block and the function will continue. Both leads to an incorrect result, but the `elif` solution will most likely blow up in one way or the other, which is preferable IMHO.

Reply:
The reason was to be able to get full test coverage. Now I tried another pattern that I've seen used: placing the check_in_list in a final else-clause, so more used to raise an error than to check the arguments beforehand.

I guess that in general there should be a preferred way, but it seems like there isn't.

Reply:
Personally, I don't worry too much about full test coverage. It's onerous in some cases, and full test coverage does not mean your code is working correctly. Test coverage is only a weak indicator and we should be pragmatic about it.

I agree there should be a preferred way. Maybe we should briefly discuss this in a dev call. Unfortunately, there is no really good solution. You always have the duplication between the if/else cases and the values in check_in_list with the danger of being out of sync. For readability, I prefer `check_in_list` at the top, because that limits the scope what can happen in the code afterwards and thus reduces mental load.